ios/brave-ios/Sources/Brave/Frontend/Browser/Toolbars/BottomToolbar/Menu/History/HistoryModel.swift

```diff
@@ -211,6 +211,15 @@ class HistoryModel: NSObject, ObservableObject {
     }
   }
 
+  @MainActor func handleRecentlyClosedSelection(_ recentlyClosed: RecentlyClosed) {
+    tabManager?.addAndSelectRecentlyClosed(recentlyClosed)
+
+    // After opening the Recently Closed in a new tab delete it from list
+    RecentlyClosed.remove(with: recentlyClosed.url)
```

Comment:
hmmm why we delete this recent closed url once we re-open it? it doesn't change the fact that this url was closed at a certain time in the past.

Reply:
Not sure actually, I copied this from the old UI (which will get deleted soon)